index.html

```diff
@@ -277,8 +277,7 @@ <h6>Project</h6>
           <li><a href="pages/devel-version.html">Development version</a></li>
           <li><a href="pages/patches.html">Patches</a></li>
           <li><a href="pages/source-code.html">Source code</a></li>
-          <li><a href="pages/contact.html#ml">Mailing lists</a></li>
-          <li><a href="pages/contact.html#gitter">Chat</a></li>
+          <li><a href="pages/contact.html">Contact</a></li>
```

Comment:
```suggestion
          <li><a href="pages/contact.html">Contacts</a></li>
```

Reply:
The page is called https://www.uyuni-project.org/pages/contact.html, and the title of the page is Contact.

I'd keep it as it is.

Reply:
As well as top menu, etc. So let's keep it as it is for now, but we can discuss a refactor about this later.